themes/LaserWave-color-theme.json

```diff
@@ -58,14 +61,30 @@
         "input.border": "#EB64B9",
         "inputOption.activeBorder": "#EB64B9",
 
-        "list.activeSelectionBackground": "#EB64B9CC",
-        "list.hoverBackground": "#EB64B9AA",
-        "list.hoverForeground": "#fff",
-        "list.inactiveSelectionBackground": "#EB64B9AA",
-        "list.inactiveSelectionForeground": "#fff",
+        "list.activeSelectionBackground": "#eb64b98f",
+        "list.activeSelectionForeground": "#eee",
+        "list.hoverBackground": "#EB64B98f",
+        "list.hoverForeground": "#eee",
+        "list.inactiveSelectionBackground": "#eb64b98f",
+        "list.inactiveSelectionForeground": "#ddd",
         "list.dropBackground": "#74dfc466",
         "list.invalidItemForeground": "#fff",
-        "tree.indentGuidesStroke": "#ffffff33"
+        "list.highlightForeground": "#eb64b9",
+        "list.focusBackground": "#91889b80",
+
+        "tree.indentGuidesStroke": "#ffffff33",
+
+        "diffEditor.border": "#b4dce7",
+        "diffEditor.removedTextBackground": "#eb64b940",
+        "diffEditor.insertedTextBackground": "#74dfc440",
+
+        "peekViewEditor.background": "#40b4c43f",
```

Comment:
I found this one a bit hard to read I would suggest taking the opacity down a bit, I thought `#40b5c449` looked pretty good

Reply:
Fixed in https://github.com/Jaredk3nt/laserwave/pull/19/commits/5534bb8f0d6532e3ed8aa0bb4bbd3c6de2013e81